src/blocks/recommendations/components/parsely-recommendations-fetcher.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { useDebounce } from '@wordpress/compose';
+import { useCallback, useEffect } from '@wordpress/element';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { setError, setRecommendations } from '../actions';
+import { useRecommendationsStore } from '../recommendations-store';
+
+const ParselyRecommendationsFetcher = ( { boost, limit, sort } ) => {
+	const {
+		state: {},
```

Comment:
SonarCloud complains about this. `state = {}` could possibly fix the complaint, but then my Intelisense complains about the variable not being used. If anyone knows of a graceful way to handle this, let me know!

Reply:
Resolved in https://github.com/Parsely/wp-parsely/pull/642/commits/cf9227c69c3e372cae8b1ed9bc3c2dd5cac21cdd by @pauarge.